sys-apps/shadow/shadow-4.14.2.ebuild

```diff
@@ -0,0 +1,270 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# Upstream sometimes pushes releases as pre-releases before marking them
+# official. Don't keyword the pre-releases!
+# Check https://github.com/shadow-maint/shadow/releases.
+
+VERIFY_SIG_OPENPGP_KEY_PATH="${BROOT}"/usr/share/openpgp-keys/sergehallyn.asc
```

Comment:
I think this needs to be updated with the new key?

Reply:
Please see the comments above

Reply:
Oh, so this is waiting for @thesamesam to create a sec-keys ebuild?

Reply:
I think @thesamesam is busy recently. But, yes, basically it's waiting for someone to create this :)
I'm not sure how to switch between the keys based on version. If I do it then I would creae a shadow-maintainers ebuild probably to not have to switch the keys :)

Reply:
Ok, I'll work on adding Alejandro's key.